bin/run

```diff
@@ -59,6 +59,7 @@ case $1 in
     ;;
   test)
     pytest tests/ missioncontrol/
+    yarn test
```

Comment:
We should run yarn lint here so we notice linting problems in CI. Can you create a small (seperate) PR to add that after this one lands?

Reply:
Actually, wait, this won't work since we don't have yarn inside our container.

For now, let's make testing (and soon linting) part of the build, by modifying `./bin/build`.